docs/guides/expectations/how_to_create_and_edit_expectations_with_a_profiler.md

```diff
@@ -9,10 +9,230 @@ This guide will help you create a new Expectation Suite by profiling your data w
+validator.save_expectation_suite(discard_failed_expectations=False)
+
+# Set up and run a Simple Checkpoint for ad hoc validation of our data 
+checkpoint_config = {
+    "class_name": "SimpleCheckpoint",
+    "validations": [
+        {
+            "batch_request": batch_request,
+            "expectation_suite_name": expectation_suite_name,
+        }
+    ],
+}
+checkpoint = SimpleCheckpoint(
+    f"_tmp_checkpoint_{expectation_suite_name}", context, **checkpoint_config
+)
+checkpoint_result = checkpoint.run()
+
+# Build and open Data Docs
+context.build_data_docs()
+
+validation_result_identifier = checkpoint_result.list_validation_result_identifiers()[0]
+context.open_data_docs(resource_identifier=validation_result_identifier)
+```
+
+And you're all set!
+
+## Optional Parameters
+
+The UserConfigurableProfiler can take a few different parameters to further hone the results. These parameters are:
+
+- `excluded_expectations`: Takes a list of expectation names which you want to exclude from the suite
+- `ignored_columns`: Takes a list of columns for which you may not want to build expectations (i.e. if you have metadata columns which might not be the same between tables
+- `not_null_only`: Takes a boolean. By default, each column is evaluated for nullity. If the column values contain fewer than 50% null values, then the profiler will add `expect_column_values_to_not_be_null`; if greater than 50% it will add `expect_column_values_to_be_null`. If `not_null_only` is set to True, the profiler will add a not_null expectation irrespective of the percent nullity (and therefore will not add an `expect_column_values_to_be_null`)
+- `primary_or_compound_key`: Takes a list of one or more columns. This allows you to specify one or more columns as a primary or compound key, and will add `expect_column_values_to_be_unique` or `expect_compound_column_values_to_be_unique`
+- `table_expectations_only`: Takes a boolean. If True, this will only create table-level expectations (i.e. ignoring all columns). Table-level expectations include `expect_table_row_count_to_equal` and `expect_table_columns_to_match_ordered_list`
+- `value_set_threshold`: Takes a string from the following ordered list - "none", "one", "two", "very_few", "few", "many", "very_many", "unique". When the profiler runs, each column is profiled for cardinality. This threshold determines the greatest cardinality for which to add `expect_column_values_to_be_in_set`. For example, if `value_set_threshold` is set to "unique", it will add a value_set expectation for every included column. If set to "few", it will add a value_set expectation for columns whose cardinality is one of "one", "two", "very_few" or "few". The default value here is "many". For the purposes of comparing whether two tables are identical, it might make the most sense to set this to "unique".
+- `semantic_types_dict`: Takes a dictionary. Described in more detail below.
```

Comment:
This is a super helpful section but I find it a little hard to read when Docusaurus renders it. Not sure if I have a great solution here though.

Also, thoughts on type hinting the optional args? Perhaps this section is more about the concepts and we can defer to the types in the code?

<img width="997" alt="Screen Shot 2021-12-17 at 2 50 09 PM" src="https://user-images.githubusercontent.com/49923762/146599945-d80452f3-ad4b-409d-b9d4-84e1b304da75.png">